Feat mesh blend

ademola-lou opened the pull request:
**Description**

A simple implementation of mesh blend based on mesh seams smoothing [https://www.jacktollenaar.top/mesh-seam-smoothing-blending#h.50wag6hqg9gh]

[https://rawcdn.githack.com/ademola-lou/three.js/refs/heads/meshblend-demo/examples/webgpu_postprocessing_meshblend.html](https://rawcdn.githack.com/ademola-lou/three.js/refs/heads/meshblend-demo/examples/webgpu_postprocessing_meshblend.html)


<img width="1521" height="1341" alt="Screenshot 2025-09-28 231147" src="https://github.com/user-attachments/assets/ce9a20c2-d31d-4c9e-9c1d-2991f7f210ee" />

Mugen87 commented on examples/jsm/tsl/display/MeshBlendNode.js:
```diff
@@ -0,0 +1,124 @@
+
+    setup(){
+         const CustomHash = wgslFn(`
+            fn Hash(p: vec3f) -> f32 {
```
Please implement all shader code as TSL. Otherwise it won't work with the WebGL backend.

ademola-lou commented:
i have removed the build folder not sure why the conflict is still showing up

Mugen87 commented:
> i have removed the build folder not sure why the conflict is still showing up

You are not supposed to remove the build files from the PR. You should only revert the related changes.

If you have issues with fixing this, I suggest you use the `reset` git command and reset all commits you have done so far. You then undo the changes to the build files and commit the remaining code as a single new commit. In this way, the PR has a clean commit history.

ademola-lou committed:
chore: squash feature changes from feat-mesh-blend (exclude build files)

ademola-lou committed:
Examples: webgpu_postprocessing_meshblend - remove unused imports and inspector nodes, tidy formatting

- remove unused three/tsl imports (add, vec3, vec4, colorToDirection, sample, depth)
- drop unused local variable (box) and unused MRT inspector nodes (diffuse, normal, velocity)
- remove unused sceneNormal sample helper and related comment
- minor formatting/spacing and string-quote cleanups for consistency

Mugen87 commented:
Thanks for your work on this but apart form the open code issues, the technique itself is quite specific and thus better placed in a separate third-party repository. Feel free to create a [community example](https://threejs.org/examples/?q=community) to demonstrate your Addon.
